.editorconfig

```diff
@@ -6,11 +6,3 @@ indent_style = space
 end_of_line = lf
 insert_final_newline = true
 charset = utf-8
-
-[*.cs]
-csharp_new_line_before_open_brace = none
```

Comment:
I got so used to .ts/.js style and found it hard to go back to wasting so much space, but I'm OK going with prevailing .NET style.

Reply:
I was about to ask why this was done this way. I do agree that with the fact that this looks much better with JS style but I guess is do we want to be consitent with Dotnet or keep our style consistent

Reply:
I know others didn't like my C# formatting. It makes sense to follow the C# defaults. At the time, my code was the only C# and I was the only one writing it, but now that we have all the other C# and the rest is in the default style, it makes perfect sense to format cadl-vs to go along with it. I'm totally fine with that.